soroban-sdk/src/storage.rs

```diff
@@ -382,6 +382,21 @@ impl Instance {
         )
         .unwrap_infallible();
     }
+
+    pub fn extend_other_contract_ttl(
```

Comment:
I think this function feels a little out of place here, but I think we can move it later on if the evolution of these modules makes another solution more obvious.

The reason I say this is because it operates on other contracts. The other functions at this level feel, to me at least, very much like they are operating on this contracts storage instance.

This is true of most of the functions under env. And we don't really have a good story for where functions for other contracts should live.

I'm surfacing this concern, but I don't think we need to address it now or soon.

Reply:
I would still rather move this just into `env`; it definitely doesn't belong to the instance storage (which is implied to belong to the current contract. I've been also thinking about moving this to deployer module, because that's the context where one would mostly likely need this function.

Reply:
That's up to debate, but I think `env.deployer().extend_ttl()` would already be much better/easier to find. We could maybe add e.g. an optional `extend_ttl` value to the `deploy` function itself, but then it would unnecessarily restrict the users who want to just bump another contract for whatever reason (which, again, seems unlikely, but possible in theory).

Reply:
`env.deployer().extend_ttl()` seems sufficient for now. Complicating the other fns, unclear on benefit.

Reply:
Moved to deployer